backend/schema/validate_test.go

```diff
@@ -379,6 +379,43 @@ func TestValidate(t *testing.T) {
 				`4:19: unexpected token "s"`,
 			},
 		},
+		{name: "InvalidPubSub",
+			schema: `
+			module test {
+				export topic topicA test.eventA
+			
+				topic topicB test.eventB
```

Comment:
I think let's format topics and subscriptions without spaces between them, they're always one-liners.

Reply:
I've moved them to one line 👍
It looks a bit crowded when theres comments attached, but if we want to fix that maybe it's a general change we could do later where we include gaps if there's comments, because that's not limited to only this decl type.
```
topic payins pubsub.PayinEvent
// publicBroadcast is a topic that broadcasts payin events to the public.
// out of order with subscription registration to test ordering doesn't matter.
export topic publicBroadcast pubsub.PayinEvent
subscription broadcastSubscription pubsub.publicBroadcast
subscription paymentProcessing pubsub.payins
```